plasma_framework/migrations/1040_setup.js

```diff
@@ -0,0 +1,60 @@
+/* eslint-disable no-console */
+const PlasmaFramework = artifacts.require('PlasmaFramework');
+const PaymentExitGame = artifacts.require('PaymentExitGame');
+const FeeExitGame = artifacts.require('FeeExitGame');
+const EthVault = artifacts.require('EthVault');
+const Erc20Vault = artifacts.require('Erc20Vault');
+const EthDepositVerifier = artifacts.require('EthDepositVerifier');
+const Erc20DepositVerifier = artifacts.require('Erc20DepositVerifier');
+const childProcess = require('child_process');
+const config = require('../config.js');
+const pck = require('../package.json');
+
+module.exports = async (
+    deployer,
+    _,
+    // eslint-disable-next-line no-unused-vars
+    [deployerAddress, maintainerAddress, authorityAddress],
+) => {
+    const vault = process.env.VAULT || false;
```

Comment:
A discussion on `VAULT`. Since we have removed the activate childchain function, so there is actually no "vault" interaction anymore, right? Could we name the flag with better naming that does not confuse. Or is the Multisig contract considered as a form of vault too?

Reply:
The idea is that you'd use the vault *with* multisig maintainer account, so they're both enabled/disabled under one flag. Perhaps if there's need, we could also have that optional...

- So you could use the vault, but no multisig maintainer.
- You could use multisig maintainer, but not vault.

Would that be better?

Reply:
I am just wondering from the perspective of this repo, it does not make interaction to "vault" after we removed the activate childchain function. So it is not really a concept needed in this repo any more. So just wonder if there is better naming we can name this flag to...idk represent we are deploying with extra security care 😅

But yeah, I don't think we should complex this to 2 flag yet unless we see the need. I am just thinking if there can have a better naming for this concept but this comment could be minor. Not a blocker comment IMO

Reply:
Not sure I understand fully ... but Vault is still used and interacted with: https://github.com/omgnetwork/plasma-contracts/pull/678/files#diff-6ea48dfd3d7c2965be57aa58597eb191R14

because it needs to create the wallet and authority account that's later used.

Reply:
oh! so actually we ... create the authority address via API during truffle migration?!
I misunderstood as we already have the authority account initiated/created beforehand.
I see

Reply:
we could do that but then ... it needs to be automated somewhere so I thought It might as well be here :)